### PR TITLE
Cookbook grammar and style fixes

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets | - |

### CODE REVIEW

File: cookbook/cache/varnish.rst [hunk 1]
```diff
@@ -192,7 +191,7 @@ Symfony adds automatically:
 .. tip::
 
     If you followed the advice about ensuring a consistent caching
-    behavior, those vcl functions already exist. Just append the code
+    behaviour, those VCL functions already exist. Just append the code
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: behavior is correct (we use American English)

File: cookbook/configuration/environments.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ How to Master and Create new Environments
 
 Every application is the combination of code and a set of configuration that
 dictates how that code should function. The configuration may define the
-database being used, whether or not something should be cached, or how verbose
+database being used, if something should be cached or how verbose
```
Comment (xabbuh) [inline cookbook/configuration/environments.rst]: the comma should be removed

Comment (javiereguiluz) [inline cookbook/configuration/environments.rst]: Are you sure that this comma is useless? The text is describing three separate things, so I find it normal that they are separated by the comma and the `or`.

Comment (xabbuh) [inline cookbook/configuration/environments.rst]: Ah, you're right. I misunderstood the sentence before (didn't read it carefully enough). Sorry for that.

File: cookbook/cache/varnish.rst (hunk 1 above)
Comment (javiereguiluz) [inline cookbook/cache/varnish.rst]: Fixed. Thanks.

**Comment by weaverryan** [comment]
Awesome stuff! Thanks to both @frne and @javiereguiluz!

**Pull request merged**